api/client/samples/similar_regions/similar_region.py

```diff
@@ -3,7 +3,8 @@
 from sklearn.neighbors import BallTree
 from api.client.batch_client import BatchClient
 from api.client.lib import get_default_logger
-from api.client.samples.similar_regions.similar_region_state import SimilarRegionState
+#from api.client.samples.similar_regions.similar_region_state import SimilarRegionState
```

Comment:
style: make all imports absolute

Reply:
This was changed to allow correct importing if multiple versions are installed on the same machine, an admittedly fringe case. I actually happen to prefer relative import in this particular case anyway simply because absolute import string is so long but if company's style is to only use absolute imports, so be it.
 Change reverted.